### DISCUSSION TITLE
Seeking interview

### DISCUSSION THREAD

**Discussion by Mattman9119** [DigiByte-Core/digibyte#92] [General]
Hey Digibyte community!

I’m am in search for someone involved in Digibyte, or someone who knows a lot about digibyte that would be willing to go on a YouTube channel for a zoom interview after the head of that channel talks with you and makes sure you are a good fit, like you know the basics and can explain everything in “newbie’s terms”.

The reason?

We just want to bring more exposure to Digibyte as we are a fan! The channel has around 200k subs and a good community itself that can help share the interview and get it out there more! Hit me up if you are interested or if you know anyone who you think would fit! Thanks!

**Comment by SmartArray** [comment]
@JaredTate ?

**Comment by j50ng** [comment]
> Hey Digibyte community!
> 
> I’m am in search for someone involved in Digibyte, or someone who knows a lot about digibyte that would be willing to go on a YouTube channel for a zoom interview after the head of that channel talks with you and makes sure you are a good fit, like you know the basics and can explain everything in “newbie’s terms”.
> 
> The reason?
> 
> We just want to bring more exposure to Digibyte as we are a fan! The channel has around 200k subs and a good community itself that can help share the interview and get it out there more! Hit me up if you are interested or if you know anyone who you think would fit! Thanks!

Hi there- Can you give us a link to the YouTube channel?
